src/openssl/algorithm.cr

```diff
@@ -0,0 +1,28 @@
+require "openssl"
+
+module OpenSSL
+  enum Algorithm
+    MD4
+    MD5
+    RIPEMD160
+    SHA1
+    SHA224
+    SHA256
+    SHA384
+    SHA512
+
+    def to_evp
+      case self
+      when OpenSSL::Algorithm::MD4       then LibCrypto.evp_md4
```

Comment:
You can skip `OpenSSL::Algorithm::` part. ditto all below

Reply:
I fixed this.